src/wap_client.c

```diff
@@ -177,7 +177,7 @@ prepare_get_command (client_t *self)
 static void
 signal_have_get_ok (client_t *self)
 {
-    zsock_send (self->cmdpipe, "sip", "GET OK", 0, 
+    zsock_send (self->cmdpipe, "sip", "GET OK", 0,
                 wap_proto_get_tx_data (self->message));
```

Comment:
How does this change relate to the "start" command whoss behaviour we're trying to change?